src/cli/update.ts

```diff
@@ -52,7 +52,9 @@ export default function (program: RootCmd) {
         console.log(`Module updated!`);
       } catch (e) {
         Log.debug(e);
-        Log.error(e.message);
+        if (e instanceof Error) {
+          Log.error(e.message);
+        }
```

Comment:
why is this necessary? can catch ever have something not an Error? it seems like just too much.

Reply:
Yeah it's a result of adding `strict` to tsconfig, just one of those fun little js things.... * sigh *